app/src/main/java/org/mozilla/focus/gecko/GeckoViewPrompt.java

```diff
@@ -50,7 +50,7 @@
 import java.util.Date;
 import java.util.Locale;
 
-final class GeckoViewPrompt implements GeckoSession.PromptDelegate {
+public final class GeckoViewPrompt implements GeckoSession.PromptDelegate {
```

Comment:
\[PMD\] INFO: The class 'GeckoViewPrompt' has a Standard Cyclomatic Complexity of 7 \(Highest = 33\)\.